std/php7/Lib.hx

```diff
@@ -85,12 +85,12 @@ class Lib {
 		return untyped __call__("fpassthru", __call__("fopen", file,  "r"));
 	}
 
-	public static function toPhpArray(a : Array<Dynamic>) : NativeArray {
-		throw "Not implemented";
+	public static inline function toPhpArray(a : Array<Dynamic>) : NativeArray {
```

Comment:
These functions should be dropped in favor of using casts (must be explained of course). But keeping them might lead to confusion.